src/Angor/Client/Pages/CheckTransactionCode.razor

```diff
@@ -130,7 +130,7 @@ else
         localAccountInfo = GetAccountInfoFromStorage();
 
         ExtPubKey accountExtPubKey = ExtPubKey.Parse(localAccountInfo.ExtPubKey, network);
-        var walletWords = _walletStorage.GetWallet().WalletWords;
+        var walletWords = new WalletWords();// this is test code we can probably delete it _walletStorage.GetWallet().WalletWords;
```

Comment:
What do you mean by test code we can probably delete?

Reply:
Check transaction code page is a test page no?
We can delete it because we made it to check we can sign a trx in wasm